app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
```

Comment:
Please sort your imports according to PEP8
You can use tools, such a isort https://pycqa.github.io/isort/

Reply:
Don't use \ to separate import in each line, pack then into a tuple: 
`from app.errors import (
NotVaccinatedError, 
OutdatedVaccineError,
 NotWearingMaskError
)`